src/forms.py

```diff
@@ -397,14 +401,15 @@ def construct_alertform(self, args):
 
             #grab the pointings instrument ids
             instrumentids = [x.instrumentid for x in pointing_info]
+            unique_instrumentids = list(set(instrumentids))
```

Comment:
The list of instrumentids contains some instruments repeated many times. Using this in a filter is inefficient.